.github/workflows/build.yml

```diff
@@ -42,6 +42,13 @@ jobs:
           toolchain: "1.85"
           override: true
 
+      - uses: Swatinem/rust-cache@v2
+        with:
+          key: "rust-cache-${{ matrix.os }}"
+          env-vars: "CARGO CC CFLAGS CXX CMAKE RUST"
+          cache-all-crates: true
+          cache-targets: false
```

Comment:
This being set to `false` I think makes this worth trying. The `target/` directory would bust the cache regularly. A single build of `core` with datafusion produces over 3GB of `.target/` data on my machine :sob: